sdk/python/feast/entity.py

```diff
@@ -97,88 +95,68 @@ def __str__(self):
 
     @property
     def name(self) -> str:
-        """
-        Gets the name of this entity.
-        """
         return self._name
 
     @name.setter
-    def name(self, name):
-        """
-        Sets the name of this entity.
-        """
+    def name(self, name: str):
         self._name = name
 
     @property
-    def description(self) -> str:
-        """
-        Gets the description of this entity.
-        """
-        return self._description
+    def value_type(self) -> ValueType:
+        return self._value_type
 
-    @description.setter
-    def description(self, description):
-        """
-        Sets the description of this entity.
-        """
-        self._description = description
+    @value_type.setter
+    def value_type(self, value_type: ValueType):
+        self._value_type = value_type
 
     @property
     def join_key(self) -> str:
-        """
-        Gets the join key of this entity.
-        """
         return self._join_key
 
     @join_key.setter
-    def join_key(self, join_key):
-        """
-        Sets the join key of this entity.
-        """
+    def join_key(self, join_key: str):
         self._join_key = join_key
 
     @property
-    def value_type(self) -> ValueType:
-        """
-        Gets the type of this entity.
-        """
-        return self._value_type
+    def description(self) -> str:
+        return self._description
 
-    @value_type.setter
-    def value_type(self, value_type: ValueType):
-        """
-        Sets the type of this entity.
-        """
-        self._value_type = value_type
+    @description.setter
+    def description(self, description: str):
+        self._description = description
 
     @property
-    def labels(self) -> Dict[str, str]:
-        """
-        Gets the labels of this entity.
-        """
-        return self._labels
+    def tags(self) -> Dict[str, str]:
+        return self._tags
 
-    @labels.setter
-    def labels(self, labels: Dict[str, str]):
```

Comment:
Isn't this a breaking change? I think we should keep backwards compat for at least one minor version.

Reply:
yeah good catch - just updated this